icu4c/source/common/ulocimp.h

```diff
@@ -61,6 +62,31 @@ ulocimp_getCountry(const char *localeID,
                    char *country, int32_t countryCapacity,
                    const char **pEnd);
 
+/**
+ * Returns a well-formed language tag for this locale ID.
+ * <p>
```

Comment:
No <p> in doxygen, please.
(And Andy just changed unistr.h from HTML to more readable, more editor-friendly Markdown such as `**bold**` rather than `<b>bold</b>`...)

Reply:
Done.